### PR TITLE
migration: modules typescript 5

### PR DESCRIPTION
**Pull request by bang9**
[CLNP-1247](https://sendbird.atlassian.net/browse/CLNP-1247?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ)

Modules

- ./modules/ChannelList/components/ChannelPreview/utils.js
- ./modules/ChannelList/components/ChannelPreviewAction.jsx
- ./modules/ChannelList/components/Placeholder.jsx
- ./modules/ChannelList/dux/actionTypes.js
- ./modules/ChannelList/dux/initialState.js
- ./modules/ChannelList/dux/reducers.js
- ./modules/ChannelList/utils.js
- (removed) ./modules/ChannelList/components/utils.js

[CLNP-1247]: https://sendbird.atlassian.net/browse/CLNP-1247?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/OpenChannelApp/components/Profile.tsx [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
 interface Props {
-  user: SendBird.User;
+  user: User;
 }
 
 export default function Profile({
   user,
 }: Props): ReactElement {
   return (
-    <div className="profile-component">
```
Comment (AhyoungRyu) [inline src/modules/OpenChannelApp/components/Profile.tsx:14]: Are these intentional changes? double -> single quote

Comment (bang9) [reply src/modules/OpenChannelApp/components/Profile.tsx:14]: It's edited by lint!

Comment (AhyoungRyu) [reply src/modules/OpenChannelApp/components/Profile.tsx:14]: Oh I thought  "jsx-quotes": ["error", "prefer-double"] has been applied to the project but I guess it's not.

**Review by AhyoungRyu — APPROVED** [review]
Regardless the tslint rule, looks great to merge!

**Pull request merged**